src/schema/v2/userPurchases.ts

```diff
@@ -16,7 +17,7 @@ export const edgeFields: Thunk<GraphQLFieldConfigMap<
   ...IDFields,
   ownerType: { type: GraphQLString, resolve: ({ owner_type }) => owner_type },
   salePrice: { type: GraphQLFloat, resolve: ({ sale_price }) => sale_price },
-  saleDate: { type: GraphQLString, resolve: ({ sale_date }) => sale_date },
+  saleDate: date(({ sale_date }) => sale_date),
```

Comment:
The weird thing is that his still returns the same type of string: `"saleDate": "2022-06-11T20:00:00+00:00",` when running the query or the test.